packages/mobile/metro.config.js

```diff
@@ -11,7 +11,9 @@ const clientPath = path.resolve(__dirname, '../web')
 const commonPath = path.resolve(__dirname, '../../packages/common')
 const harmonyPath = path.resolve(__dirname, '../../packages/harmony')
 const splPath = path.resolve(__dirname, '../../packages/spl')
-const sdkPath = path.resolve(__dirname, '../../packages/libs')
+const libsPath = path.resolve(__dirname, '../../packages/libs')
+const sdkPath = path.resolve(__dirname, '../../packages/sdk')
+const ethPath = path.resolve(__dirname, '../../packages/eth')
```

Comment:
surprised this was needed

Reply:
I think metro doesn't follow the * deps like the rest, so all of them need to be specified?